.circleci/config.yml

```diff
@@ -1,40 +1,20 @@
-version: 2
+version: 2.1
+
+orbs:
+  ft-golang-ci: financial-times/golang-ci@1
+
 workflows:
-  version: 2
-  build-docker-image:
+  tests_and_docker:
     jobs:
-      - build:
-          filters:
-            tags:
-              only: /.*/
-jobs:
-  build:
-    docker:
-      - image: docker:stable-git
-    steps:
-      - checkout
-      - setup_remote_docker
-      - run:
-          name: Set Docker image tag based on branch or git tag
-          command: |
-            if [ ! -z "${CIRCLE_TAG}" ] ; then
-              echo 'export DOCKER_TAG=${CIRCLE_TAG}' >> $BASH_ENV
-            elif [ "${CIRCLE_BRANCH}" == "master" ] ; then
-              echo 'export DOCKER_TAG=latest' >> $BASH_ENV
-            else
-              echo 'export DOCKER_TAG=$( echo ${CIRCLE_BRANCH} | cut -f 2 -d / )' >> $BASH_ENV
-            fi
-      - run:
-          name: Build Docker image
-          command: |
-            source ${BASH_ENV}
-            docker build \
-            --build-arg SONATYPE_USER=${SONATYPE_USER} \
-            --build-arg SONATYPE_PASSWORD=${SONATYPE_PASSWORD} \
-            --tag nexus.in.ft.com:5000/coco/${CIRCLE_PROJECT_REPONAME}:${DOCKER_TAG} .
-      - deploy:
-          name: Push Docker image
-          command: |
-            source ${BASH_ENV}
-            docker login nexus.in.ft.com:5000 -u ${DOCKER_USER} -p ${DOCKER_PASSWORD}
-            docker push nexus.in.ft.com:5000/coco/${CIRCLE_PROJECT_REPONAME}:${DOCKER_TAG}
\ No newline at end of file
+      - ft-golang-ci/build-and-test:
+          name: build-and-test-project
+      - ft-golang-ci/docker-build:
+          name: build-docker-image
+          requires:
+            - build-and-test-project
+  snyk-scanning:
+    jobs:
+      - ft-golang-ci/scan:
+          name: scan-dependencies
+          context: cm-team-snyk
+            
```

Comment:
Make LE consistent, please fix this one.

Reply:
Done